zipkin-lens/src/components/DependenciesPage/DependenciesPageHeader.jsx

```diff
@@ -87,6 +90,21 @@ const DependenciesPageHeader = React.memo(
     const classes = useStyles();
     const { i18n } = useLingui();
 
+    const dateTimePickerCommonProps = React.useMemo(
+      () => ({
+        inputVariant: 'outlined',
+        className: classes.dateTimePicker,
+        InputProps: {
+          // See: https://github.com/openzipkin/zipkin/issues/3052
+          // DateTimePicker's input is not very useful, so disable it and
+          // let users use a calendar dialog instead.
+          disabled: true,
```

Comment:
I think this is fine for now but I'll probably look into whether it's possible to make it not barf at some point. On desktop, I usually get frustrated when a date input doesn't let me type in the date since it's usually way faster than scrolling through the calendar.